client/README.md

```diff
@@ -12,4 +12,4 @@ and external templates including:
 * Quick info
 * Go to definition
 
-This extension uses `@angular/language-service@5.0.0-beta.5` and `typescript@2.4.2`.
+This extension uses `@angular/language-service@6.0.0-beta.1-8df56fe` and `typescript@2.6.2`.
```

Comment:
the hash doesn't seem to match https://github.com/angular/angular/releases/tag/6.0.0-beta.1 ?

Reply:
Fixed.